source/getting-started.txt

```diff
@@ -0,0 +1,63 @@
+.. _app-services-get-started:
+
+================
+Getting Started
+================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Template Apps </template-apps>
+   Tutorials </tutorials>
+
+The resources on this page are designed to help you explore Atlas App 
+Services. Visit the Template Apps page to bootstrap an App Services app. 
+Check out tutorials to learn more about features, such as Device Sync 
+and client authentication.
+
+Template Apps
+-------------
+
+Bootstrap App Services with a Template App. Template apps bring together
+many of the building blocks available in Atlas App Services and start you 
+off with a prebuilt app that you can customize. `Learn more about the Template
```

Comment:
It might be cool to give readers an idea of the types of App Services apps they can bootstrap here. Maybe move the "Learn more about the Template Apps" line down below a bullet list showing something like:

> Template Apps provide pre-built solutions that implement:
> - A mobile todo app written with Realm SDKs that syncs to an App Services App
> - A hosted todo web app using the GraphQL API or Realm Web SDK
> - An Event-driven [Database Trigger](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-24302/triggers/database-triggers/#std-label-database-trigger) template that updates a view in a separate collection.
>
> Learn more about the Template Apps...